### PR TITLE
Add prerequisites info to installation instructions. Add headers for install.

### PR DESCRIPTION
**Pull request by jascott1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,26 @@ We use images based on a Kolla build, but we've dropped their Ansible approach t
...
+
+**Kubernetes with Helm**
+
+See https://github.com/kubernetes/kubernetes and https://github.com/kubernetes/helm
+
+**Local Charts server**
+
+During installation, openstack-helm expects a chart server running on localhost. You can start one with the following command:
+
+    helm serve --address localhost:8879
```
Comment (databus23) [inline README.md:54]: I think the `--address localhost:8879` is redundant as its the default behaviour of `helm serve`

Comment (jascott1) [reply README.md:54]: Helm appears to serve on 127.0.0.1 by default and it did not match localhost as specified in the charts' requirements.yml

Comment (databus23) [reply README.md:54]: I don't get the difference between 127.0.0.1 and localhost in this case, it should be the same. I just to `helm serve` and using the makefile in this repo works for me. What error are you getting when you run it without the `--address localhost:8879`

**Comment by fwiesel** [comment]
Closing as the project is deprecated in favour of sapcc/helm-charts

**Pull request closed without merging**